apps/bot-discord/src/config.json

```diff
@@ -9,24 +9,101 @@
     "host": "127.0.0.1"
   },
   "responses": [
+    {
+      "label": "revanced_crash",
+      "threshold": 0.85,
+      "reply": {
+        "title": "Why is my patched app crashing?",
+        "description": "It seems like your patched application is crashing, these might be the reason why:\n  * You patched an unrecommended version.\n  * Patches failed to apply.\n   * Unrecommended patches.\n  * Split APK was used.",
```

Comment:
You should change "Unrecommended patches" to "Non default patches". Do this change for TG too.